eclair-core/src/main/scala/fr/acinq/eclair/router/Router.scala

```diff
@@ -526,6 +561,21 @@ object Router {
     blockHeight < staleThresholdBlocks && update1_opt.map(isStale).getOrElse(true) && update2_opt.map(isStale).getOrElse(true)
   }
 
+  /**
+    *
+    * @param shortChannelId short channel id
+    * @param channels channel announcements, indexed by short channel id
+    * @param updates channel updatesm indexed by channel descriptor
+    * @return true if shortChannelId is stale
+    */
+  def isStale(shortChannelId: ShortChannelId, channels: Map[ShortChannelId, ChannelAnnouncement], updates: Map[ChannelDesc, ChannelUpdate]): Boolean = {
```

Comment:
not sure why we need this function, why doesn't the wip-routing-sync branch need it?